src/domain/ws/observable.ts

```diff
@@ -0,0 +1,39 @@
+type Func = (...args: any[]) => any;
```

Comment:
this looks like coming from here.

https://github.com/nimiq/electrum-client/blob/master/electrum-ws/ElectrumWS.ts

There is a way we can use that library directly? and/or fork to change Liquid responses/data structure?


We should make attribution of code if not

Reply:
Yeah it's a copy paste of the web socket client. We cannot use it due to weird dependency with bitcoinjs. I think easier to just reuse that TS client only.